diff --git a/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js b/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
index 00ae69d..035126f 100644
--- a/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
+++ b/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
@@ -93,6 +93,8 @@ exports.networks = [
     { chainId: 1101n, shortName: 'zkevm' },
     { chainId: 1111n, shortName: 'wemix' },
     { chainId: 1112n, shortName: 'twemix' },
+    { chainId: 11124n, shortName: 'abstract-sepolia' },
+    { chainId: 2741n, shortName: 'abstract' },
     { chainId: 1115n, shortName: 'tcore' },
     { chainId: 1116n, shortName: 'core' },
     { chainId: 1135n, shortName: 'lisk' },
